src/Tasks/System.Resources.Extensions.pkgdef

```diff
@@ -0,0 +1,7 @@
+[$RootKey$\RuntimeConfiguration\dependentAssembly\bindingRedirection\{1A1A9DA4-9F25-4AB8-89BF-BCEF73875CA8}]
+"name"="System.Resources.Extensions"
+"codeBase"="$BaseInstallDir$\MSBuild\Current\Bin\System.Resources.Extensions.dll"
+"publicKeyToken"="cc7b13ffcd2ddd51"
+"culture"="neutral"
+"oldVersion"="0.0.0.0-99.9.9.9"
```

Comment:
Please don't redirect references to anything newer than `newVersion`. This should be `0.0.0.0-4.0.0.0`.
Otherwise if folks want to use 5.0.0.0 and ship that version, you'll cause them to malfunction by forcing them into the 4.0.0.0 world.